request.js

```diff
@@ -108,6 +140,12 @@ function Request (options) {
     options = self._har.options(options)
   }
 
+  // transform `formData` for backward compatibility
+  // don't check for explicit object type to support legacy shenanigans
+  if (options.formData && !Array.isArray(options.formData)) {
```

Comment:
`options.formData &&` can be removed.

Reply:
I think code is right here. The array check is done only if formData is present.